include/signalrclient/web_exception.h

```diff
@@ -12,8 +12,16 @@ namespace signalr
     class web_exception : public std::runtime_error
     {
     public:
-        explicit web_exception(const utility::string_t &what)
-            : runtime_error(utility::conversions::to_utf8string(what))
+        web_exception(const utility::string_t &what, unsigned short status_code)

```

Comment:
I'm assuming we don't have the same problem with public api changes as the other projects?